doc-content/enterprise-only/openshift/rhpam72-prod-immutable-monitor-ref.adoc

```diff
@@ -257,8 +264,8 @@ for more information.
 |`SSO_PASSWORD` | RH-SSO Realm Admin Password used to create the Client | `${SSO_PASSWORD}`
 |`SSO_DISABLE_SSL_CERTIFICATE_VALIDATION` | RH-SSO Disable SSL Certificate Validation | `${SSO_DISABLE_SSL_CERTIFICATE_VALIDATION}`
 |`SSO_PRINCIPAL_ATTRIBUTE` | RH-SSO Principal Attribute to use as username. | `${SSO_PRINCIPAL_ATTRIBUTE}`
-|`HOSTNAME_HTTP` | Custom hostname for http service route.  Leave blank for default hostname, e.g.: <application-name>-rhpamcentrmon-<project>.<default-domain-suffix> | `${BUSINESS_CENTRAL_HOSTNAME_HTTP}`
```

Comment:
I think these changes are a mistake. This looks to me like it is 4.1.2.3.3.6. Image Environment Variables in the rendered document which is listing env vars for rhpamcentrmon. The description shouldn't be referencing smartrouter.

Reply:
The changes are in the autogenerated document. I am really not comfortable making such changes apart from autogeneration. If you can not accept this, we need an urgent meeting with @dward and/or @fspolti @spolti (not sure which ref will work) to work out why the autogenerated document is like this.